website/src/pages/blog.tsx

```diff
@@ -69,6 +69,7 @@ export const pageQuery = graphql`
                     fields {
                         slug
                     }
+                    fileAbsolutePath
```

Comment:
Why do you need to request `fileAbsolutePath` this in this file?

Reply:
This was the easiest way to derive the blog post file name without adding a new field

Reply:
Good catch, yeah, not needed.